src/cmdlib.sh

```diff
@@ -264,9 +264,13 @@ commit_overlay() {
         --timestamp "${git_timestamp}"
 }
 
-runcompose() {
-    # Implement support for automatic local overrides:
-    # https://github.com/coreos/coreos-assembler/issues/118
+# Implement support for automatic local overrides:
+# https://github.com/coreos/coreos-assembler/issues/118
+#
+# This function commits the contents of overlay.d/ as well
+# as overrides/{rootfs} to OSTree commits, and also handles
+# overrides/rpm.
+prepare_compose_overrides() {
```

Comment:
Minor bikeshed: since it handles src config `overlay.d/` too, which is not an override, maybe `prepare_compose_overlays_and_overrides()`?

Reply:
That's...long.  Admittedly for a function only called once but still.
I went with `prepare_compose_overlays` since I think of "override" as a special case of "overlay", and we already have a `commit_overlay` function.